{"uuid": "49238eaf-1c09-45df-a15f-8899e7616619", "children": ["3c007e4c-37b5-4622-89c3-033472239577"], "befores": [{"name": "create_comment", "status": "broken", "statusDetails": {"message": "KeyError: 'id'\n", "trace": "  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 910, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/todo_api/conftest.py\", line 114, in create_comment\n    id_comment_created = response[\"body\"][\"id\"]\n                         ~~~~~~~~~~~~~~~~^^^^^^\n"}, "start": 1712608470940, "stop": 1712608471266}], "start": 1712608470940, "stop": 1712608471281}